main.raku

```diff
@@ -75,6 +75,7 @@ my LakeResponse $lakes = unmarshal($content, LakeResponse);
 
 my @messageItems = ();
 # filter lakes for feature: temperature
+@messageItems.push: "Aktuelle Wassertemperaturen:"
```

Comment:
This won't compile, `;` is missing at the end of the line

Reply:
Whoops. I may have skipped the tooling installation